docs/dev-guide/deploying-merlin/local_development.md

```diff
@@ -209,20 +209,20 @@ MLP and Merlin use Google Sign-in to authenticate the user to access the API and
 
 ```bash
 export OAUTH_CLIENT_ID="<put your oauth client id here>"
+export MLP_CHART_VERSION=0.4.18
 
 kubectl create namespace mlp
 
-git clone git@github.com:caraml-dev/mlp.git
-
-helm install mlp ./mlp/chart --namespace=mlp --values=./mlp/chart/values-e2e.yaml \
-  --set mlp.image.tag=main \
-  --set mlp.apiHost=http://mlp.mlp.${INGRESS_HOST}.nip.io/v1 \
-  --set mlp.oauthClientID=${OAUTH_CLIENT_ID} \
-  --set mlp.mlflowTrackingUrl=http://mlflow.mlp.${INGRESS_HOST}.nip.io \
-  --set mlp.ingress.enabled=true \
-  --set mlp.ingress.class=istio \
-  --set mlp.ingress.host=mlp.mlp.${INGRESS_HOST}.nip.io \
-  --set mlp.ingress.path="/*" \
+helm repo add caraml https://caraml-dev.github.io/helm-charts
```

Comment:
Should we make a note in this section that MLP can also be installed by the Merlin chart directly?

Reply:
I updated both local_development.md and quick_install.sh